package/src/components/Channel/Channel.tsx

```diff
@@ -1382,8 +1382,6 @@ const ChannelWithContext = <
         } else {
           updateMessage(messageResponse.message);
         }
-
```

Comment:
Is this no more needed?

Reply:
It is no longer needed, no; the optimistic update is already done beforehand.

Reply:
Okay